packages/gatsby/src/experiments.ts

```diff
@@ -0,0 +1,25 @@
+const activeExperiments = [
+  {
+    name: `FAST_DEV`,
```

Comment:
The initial set of experiments is from the dev speedup work — we'll add/remove from here as we add/graduate/pull out experiments